solr/core/src/java/org/apache/solr/api/ContainerPluginsRegistry.java

```diff
@@ -239,8 +238,7 @@ private static String getActualPath(ApiInfo apiInfo, String path) {
 
   @SuppressWarnings({"rawtypes", "unchecked"})
   private static  Map<String, String> getTemplateVars(PluginMeta pluginMeta) {
-    Map result = makeMap("plugin-name", pluginMeta.name, "path-prefix", pluginMeta.pathPrefix);
-    return result;
+    return (Map) makeMap("plugin-name", pluginMeta.name, "path-prefix", pluginMeta.pathPrefix);
```

Comment:
Should replace the call to `makeMap` by `Map.of(...)`. This allows removing the cast and removing the `@SuppressWarnings`.